### ISSUE TITLE
Cypress tests fail with default templates

### ISSUE THREAD

**Issue by robyedlin** [vuejs/create-vue#24]
### Steps to reproduce
Create a new project with Cypress > yes
Via `npm init vue@latest`

### Problem 1
Running `npm run test:unit:ci` fails

Details:
```
[vite] Internal server error: Cannot read property 'uid' of undefined
  Plugin: vite:import-analysis
  File: ../with-tests/src/components/__tests__/HelloWorld.spec.js
```
### Problem 2
Running `npm run test:e2e:ci` fails with hanging server

Details:
- No files are built in `/dist` folder, thus server returns CANNOT GET /

**Comment by sodatea** [comment]
Cannot reproduce.
1. Do you have a reproduction repo?
2. What's your OS and Node.js version?

**Comment by sodatea** [comment]
For problem 2, you need to run `npm run build` before `npm run test:e2e:ci`.

**Comment by robyedlin** [comment]
Problem 1: I'm using github codespaces, which is node 14 and linux. I will add a repo, but it's the repo generated by `npm init vue@latest`
Problem 2: Yes, that's what I'm currently doing. But shouldn't `npm run build` be part of the test:e2e:ci script then, so that it won't hang? I can submit a PR for that.

**Comment by robyedlin** [comment]
@sodatea https://github.com/robyedlin/vue3-starter/blob/master/README.md

**Comment by sodatea** [comment]
The cause is similar to this: https://github.com/browserify/resolve/issues/264#issuecomment-1006948040
Cypress has bundled a problematic version of `graceful-js` and its polyfill breaks the newest version of `resolve`.

**Comment by sodatea** [comment]
I've reported it to Cypress: https://github.com/cypress-io/cypress/issues/19610

**Comment by sodatea** [comment]
> Problem 2: Yes, that's what I'm currently doing. But shouldn't `npm run build` be part of the `test:e2e:ci` script then, so that it won't hang? I can submit a PR for that.

I'm not sure about this.

Sometimes users may want to change the Cypress configuration and rerun the e2e tests without rebuilding.
Considering `npm run build` is very costly, I think the two steps should be separate.

**Comment by robyedlin** [comment]
That's fair. Can I submit a PR for the docs then to make that clear?

**Comment by sodatea** [comment]
Fixed in Cypress 9.3.0
